lib/Breadcrumb/styles.scss

```diff
@@ -75,7 +79,7 @@ $breadcrumb-spacing: $layout-spacing-base !default;
       display: block;
     }
 
-    &:after {
+    &::after {
```

Comment:
What's with this rule? is there any benefit to adding two colons?

Reply:
I think you could use single colons for psudo-elements originally, but as of the CSS3 spec we should use double-colon selectors for pseudo-elements like `first-letter`, `before` and `after`, and
single-colon selectors for pseudo-selectors such as `nth-child`.

We just used the standard stylelint ruleset for SCSS and it follows the CSS3 spec.
That said, most modern browsers accept both syntaxes without issue anyway.

Reply:
Ah cool, thanks for the explainer :)